src/documents/ChangeApplierAPI.md

```diff
@@ -62,12 +62,17 @@ The 4 examples presented in the _"Examples"_ column are parallel for the two cas
 
 #### Model Listener Declaration ####
 
-A model listener declaration block has exactly the same form and meaning as any of the record types supported by [Invokers](Invokers.md) and [Listeners](EventInjectionAndBoiling.md#listener-boiling) - including the one-string compact syntax documented with [Invokers](Invokers.md). The only difference is that an extra [context name](Contexts.md) is available in this block by the name of change. This is bound to the particular change event which triggered this listener. This context behaves as an object with the following fields:
+A model listener declaration block has exactly the same form and meaning as any of the record types supported by [Invokers](Invokers.md) and [Listeners](EventInjectionAndBoiling.md#listener-boiling) - including the one-string compact syntax documented with [Invokers](Invokers.md). There are two extra features that are supported in
```

Comment:
Should the second link to "Invokers" be directly to the compact syntax?
https://github.com/amb26/infusion-docs/blob/FLUID-5489/src/documents/Invokers.md#compact-format